.github/workflows/ci_docs.yml

```diff
@@ -3,16 +3,27 @@ on:
   push:
     # Run only on changes on these files
     paths:
-      - 'lib/**.nim'
+      - 'compiler/docgen.nim'
+      - 'compiler/renderverbatim.nim'
       - 'doc/**.rst'
-      - 'doc/nimdoc.css'
+      - 'doc/nimdoc.css' # xxx is it worth duplicating just because of this entry which rarely changes?
```

Comment:
The thing is that you can't not duplicating this list :P So we just leave the entry out since it shouldn't be in anyway.

Reply:
i guess the only way is to generate those ci files in nim using `strutils.%` or `strformat.fmt`; there's actually quite a lot of duplication amongst all the yml files (eg `github/workflows/{*.yml}`) which is always a pain (you update one and forget to update the other etc).
it would actually be quite trivial to do but not sure if that'd be accepted...

Reply:
It's not the level of duplication that requires spinning up a program to generate. I'd say we're perfectly fine as is.